presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -149,6 +149,7 @@
     public static final String PARTITIONING_PRECISION_STRATEGY = "partitioning_precision_strategy";
     public static final String EXPERIMENTAL_FUNCTIONS_ENABLED = "experimental_functions_enabled";
     public static final String USE_LEGACY_SCHEDULER = "use_legacy_scheduler";
+    public static final String DISALLOW_LARGE_BYTECODE_GENERATION = "disallow_large_bytecode_generation";
```

Comment:
I would have gone with ERROR_ON since that is the exact behavior (it generates an error). Block and disallow are slightly less clear about what happens.

Reply:
done